tests/unit/contexts/test_application.py

```diff
@@ -11,18 +12,28 @@
 
 
 @pytest.mark.parametrize(
-    "attribute", ["dependency_resolver", "pip_resolver", "site_packages_path", "notebook_loader", "folder_loader"]
+    "attribute",
+    [
+        "dependency_resolver",
+        "pip_resolver",
+        "site_packages_path",
+        "notebook_loader",
+        "folder_loader",
+        "workflow_linter",
+        "used_tables_crawler_for_paths",
+        "used_tables_crawler_for_queries",
+    ],
 )
-def test_global_context_attributes_not_none(attribute: str):
+def test_global_context_attributes_not_none(attribute: str) -> None:
     """Attributes should be not None"""
```

Comment:
should not be None ?

Reply:
We add `-> None` to tests to trigger the linter. Actually not necessarily in this case as attribute has a type hint, but more consistent with other tests